src/win/fsatracedll.c

```diff
@@ -19,6 +22,24 @@ INT APIENTRY DllMain(HMODULE hDLL, DWORD Reason, LPVOID Reserved) {
 	switch (Reason) {
 	case DLL_PROCESS_ATTACH:
 		emitInit();
+		{
+			// DLLs that were loaded before we got hooked, so mark them as a read dependency
+			DWORD cb = 0;
+			wchar_t winBuf[PATH_MAX];
+			char utfBuf[PATH_MAX];
+			HANDLE hProcess = GetCurrentProcess();
+			if (EnumProcessModules(hProcess, NULL, 0, &cb)) {
+				HMODULE* modules = malloc(cb);
```

Comment:
I can't remember if this is still the case, but IIRC I tried to avoid allocating memory and linking libc (at least when debug tracing was disabled). Could this be statically allocated on the stack with a reasonable upper size? 8k entries (=64Kb stack) seems seasonable...

Reply:
That should be easy enough. Not at a computer tomorrow but I'll send a revision Friday

Reply:
The rationale is that some processes don't link to libc and I don't want to inject an artificial dependency if it can be avoided.

Reply:
Makes sense. I switched to an 8K buffer